src/main/resources/config.properties

```diff
@@ -21,4 +22,11 @@ build.workspace.computer.not.found = Unable to access the computer for this buil
 matlab.command.build.step.name = runMATLABCommand
 matlab.tests.build.step.name = runMATLABTests
 matlab.command.step.display.name = Run MATLAB commands, scripts, or functions
-matlab.tests.step.display.name = Run MATLAB tests and generate artifacts
\ No newline at end of file
+matlab.tests.step.display.name = Run MATLAB tests and generate artifacts
+matlab.custom.location = Custom...
+matlab.added.to.path.from = Prepending MATLAB to PATH variable from this location: %s
+Axis.matlab.key = MATLAB
+Axis.use.matlab.warning = This project specifies MATLAB versions using the added 'MATLAB' axis as well as the 'Use MATLAB version' option. The values specified by 'Use MATLAB version' take precedence over the values specified by the 'MATLAB' axis.
+Axis.no.installed.matlab.error = Because the project does not have any MATLAB versions configured in Jenkins Global Tool Configuration, the plugin runs the default configuration.
```

Comment:
Reading the first clause, I think we can summarize it this way. Is either of these statements accurate?

-Because the project does not use any MATLAB tools, ...
-Because no MATLAB tools have been specified, ...

I also think it makes sense here to replace "runs" with "will run", because the plugin's action will definitely take place in the future.

Reply:
I think using versions instead of tools is a bit more accurate. MATLAB is a tool that has different versions that can be configured.

Using the second suggested clause, I propose the following revision:

Axis.no.installed.matlab.error = Because no MATLAB versions are configured in Jenkins Global Tool Configuration, the plugin will run the default matrix configuration.

I added '...matrix configuration.' to emphasize the difference in Global tool configuration and a Matrix configuration.